src/arg.c

```diff
@@ -125,10 +125,12 @@ static void dcc_note_compiled(const char *input_file, const char *output_file)
  * The copy is dynamically allocated and the caller is responsible for
  * deallocating it.
  *
+ * If arguments has "-fprofile-arcs -ftest-coverage" hasgcov set to 1 or default0.
```

Comment:
Should have space between 'default' and '0'.
Need to consider the parameter '--coverage' also.

Reply:
Thank you for point out the issues. Now it fixed.